### PR TITLE
Migrate installing/enabling Chrony from CSE to VHD

### PR DESCRIPTION
**Pull request by anujmaheshwari1**
Moving: disabling time sync services NTP and timesyncd and enabling Chrony
from: CSE
to: VHD (install dependencies)

### CODE REVIEW

**Review by adilliadil — APPROVED** [review]
LGTM

**Comment by adilliadil** [comment]
@xuto2 & @anujmaheshwari1 reckon we remove the cse installation as well, or check to install it if not exists?

File: parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh [hunk 1]
```diff
@@ -91,9 +91,15 @@ disableNtpAndTimesyncdInstallChrony() {
     # Disable systemd-timesyncd
```
Comment (xuto2) [inline parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh:91]: let's move this method to tools_install_ubuntu.sh

File: parts/linux/cloud-init/artifacts/cse_main.sh [hunk 2]
```diff
@@ -52,12 +52,6 @@ fi
 configureHTTPProxyCA
 {{- end}}
 
-{{- if EnableChronyFor1804}}
```
Comment (xuto2) [inline parts/linux/cloud-init/artifacts/cse_main.sh:55]: A follow up after this is to remvoe this EnableChronyFor1804 flag from AgentBaker and RP.

File: parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh (hunk 1 above)
Comment (xuto2) [reply parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh:91]: might also be able to move the error constant from cse_helper.sh to tools_install_ubutu.sh.

ERR_STOP_OR_DISABLE_SYSTEMD_TIMESYNCD_TIMEOUT
ERR_STOP_OR_DISABLE_NTP_TIMEOUT
ERR_CHRONY_INSTALL_TIMEOUT

**Review by xuto2 — APPROVED** [review]

**Pull request merged**